app.py

```diff
@@ -31,6 +31,10 @@
     versioned_metrics.subscribe(alert_cor)
     alert_cor.subscribe(print)
 
+    # Git version alert correlation
+    version_alert_corr = flatliners.GitVersionAlertCorrelation()
+    versioned_metrics.subscribe(version_alert_corr)
```

Comment:
wouldnt it be easier to subscribe to `alert_cor`. Like we do with the `std_dev` above?
To me it seems more natural to create the correlation for a cluster and the from all clusters create one for the version.
But you can do this as a refactoring later if you want